tests/io/v3/vm/permissions/test_schema.py

```diff
@@ -0,0 +1,12 @@
+'''
```

Comment:
Change filename to `test_permissions_schema.py`
Reference: https://github.com/tenable/pyTenable/pull/502/files

Reply:
done